### ISSUE TITLE
Involuntary vocalizations

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4790]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
Involuntary vocalizations, groaning, grunting, humming included

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by MickeySegal** [comment]
We are merging our finding with one for which we use https://hpo.jax.org/app/browse/term/HP:0100035

**Comment by pnrobinson** [comment]
OK!
